### PR TITLE
added adding new messages in dialogs.

### PR DESCRIPTION
**Pull request by Aretemal**

### CODE REVIEW

File: src/components/Dialogs/Message/Message.jsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import {NavLink} from "react-router-dom";
 import classes from './../Dialogs.module.css';
 const Message = (props) => {
+    console.log('332')
```
Comment (ksavon) [inline src/components/Dialogs/Message/Message.jsx:5]: Please remove debug string

File: src/redux/state.js [hunk 2]
```diff
@@ -51,19 +53,34 @@ let store = {
...
             this._callSubscriber(this._state);
-        }
 
+
+        } else if (action.type === ADD_MESSAGE) {
+            let newMessage = {
+                id: this._state.dialogsPage.messages.length + 1,
+                message: this._state.dialogsPage.newMessageText,
+            };
+            this._state.dialogsPage.messages.push(newMessage);
```
Comment (ksavon) [inline src/redux/state.js:68]: Does `this._state` private property? If 'yes', then is there any other way to update state?

**Comment by Konstantsin** [comment]
Couple things to start using:
- follow convention when writing code. Google for "eslint and react" for verification your code.
- follow convention when writing commit message. Google for "git commit conventions" for propper naming commits.

**Comment by Aretemal** [comment]
Removed debug string. "this._state" is private property of store. "this._state" changed to "this.getState()".

**Comment by Aretemal** [comment]
added EsLint.

**Pull request merged**